week-8/Homework/mandatory/3-dog-photo-gallery/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>Random Dog</title>
+        <meta name="Random Dog" content="Random Dog">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="dog.css">
+    </head>
+    <body>
+<div class="container">
```

Comment:
Just like how we keep JS indented for readability, we should do the same for HTML.
In this case, the div is a child of body, so the whole block should be indented 1 tab further than body.